collect_app/src/main/res/layout/hierarchy_group_item.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:paddingVertical="12dp"
```

Comment:
> Should we pull out a shared dimension for the 12dp value? It looks like we're using it in quite a few places?

Yeah, I was thinking about it as well... I'm not sure about naming though, the value is between `margin_small` (8dp) and `margin_standard` (16dp) so maybe `margin_medium`?

> I'm also wondering how this ends up being so consistent with the actual question view? Do we end up with other margins adding up to 12dp? If so, maybe we can clean that up so we just end up with one view declaring the spacing.

It's 12 because in odk_view we have the group name with `margin_top` = 8dp and then below question widgets with `margin_top` = 4dp, 8+4=12 so that's 12 is the value we needed to use in the hierarchy to display the question title in the same position. Of course it would be better to use the standard margin 16dp but it's too much and and doesn't look good.

Reply:
Done.